azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc

```diff
@@ -0,0 +1,349 @@
+
+. In the *Mule Palette* view, search for `azure` and select the *Create/Rename Path* operation.
+. Drag the *Create/Rename Path* operation from the Studio canvas onto the *Listener* flow.
+. In the *Create/Rename Path* configuration, click *+* next to the *Connector Configuration* field to add a connection configuration.
+. On the *General* tab, in *Connection*, select the authentication method to configure and enter all the required values:
++
+image::azure-data-lake-create-file-system-filling-values-1.png[Create File System Config with Shared Access Signature connection values]
++
+. Click *OK*.
+. In the *File System* field, enter the value for your target file system. +
+If the file system does not exist, an error is thrown.
+. In *Path*, enter your desired path name.
+. In *Resource*, enter either `file` or `directory`:
++
+image::azure-data-lake-create-path-filling-values.png[General properties configuration for Create/Rename Path operation]
++
+. Save the project.
+. Test the app by sending a GET request to `localhost:8081/create-path`.
+
+[[create-path-xml]]
+=== XML for Creating a Path
+
+Paste this code into the *Configuration XML* tab of your flow to quickly load the flow for this example into your Mule app:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:azure-data-lake-storage="http://www.mulesoft.org/schema/mule/azure-data-lake-storage"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="
+		http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+		http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+		http://www.mulesoft.org/schema/mule/azure-data-lake-storage http://www.mulesoft.org/schema/mule/azure-data-lake-storage/current/mule-azure-data-lake-storage.xsd">
+	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config">
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<azure-data-lake-storage:config name="Azure_Data_Lake_Storage_Connector_Config" doc:name="Azure Data Lake Storage Connector Config">
+		<azure-data-lake-storage:shared-access-signature-connection accountName="${account.name}" dnsSuffix="${dns.suffix}" sasToken="${sas.token}" />
+	</azure-data-lake-storage:config>
+	<flow name="create-path-flow">
+		<http:listener doc:name="Listener" config-ref="HTTP_Listener_config" path="create-path"/>
+		<azure-data-lake-storage:create-or-rename doc:name="Create/Rename Path" config-ref="Azure_Data_Lake_Storage_Connector_Config" fileSystem="newfilesystem" path="newpath" resource="directory"/>
+	</flow>
+
+</mule>
+----
+
+[[update-path-example]]
+== Configure Update Path Operations with Actions
```

Comment:
"with Actions" sounds weird, Azure documentation refers to this operation as a plain Update Path, and has no mention of actions (it's just the name of the parameter)